src/core/getters/route.ts

```diff
@@ -1,17 +1,12 @@
+import { camel } from '../../utils/case';
 import { sanitize } from '../../utils/string';
 
 const getRoutePath = (path: string) => {
-  return path.split('').reduce((acc, letter) => {
-    if (letter === '{') {
-      return acc + '${';
-    }
-
-    if (letter === '}') {
-      return acc + '}';
-    }
-
-    return acc + sanitize(letter, { dot: true });
```

Comment:
We were allowing dot here but it can't work since in `params.ts` we wasn't allowing it 🤷

Reply:
dot are not for the params

Reply:
you don't handle the case where you have something like this /{petId}.js

Reply:
Indeed, I rewrite that part and I added jest to be able to have unit tests for this, much simpler to test and make it robust imho.